scripts/validator/index.ts

```diff
@@ -128,23 +133,32 @@ class Validator {
    */
   public async setCommission() {
     console.log(`\nSetting reward commission`);
+    const commission = +process.env.REWARD_COMMISSION * 10 ** 7;
```

Comment:
Move all magic numbers to constants.

Reply:
@AndreiNavoichyk  This is a constant value, it is not going to change.

Reply:
Moved to env